docs/platforms/javascript/common/user-feedback/configuration/index.mdx

```diff
@@ -77,70 +85,107 @@ feedbackIntegration({
-| `inputForeground`         | `--input-foreground`           | `inherit`                                 | `inherit`                                 | Foreground color for form inputs                                                                      |
-| `inputBorder`             | `--input-border`               | `var(--border)`                           | `var(--border)`                           | Border styles for form inputs                                                                         |
-| `inputOutlineFocus`       | `--input-outline-focus`        | `rgba(108, 95, 199, 1)`                   | `rgba(108, 95, 199, 1)`                   | Outline color for form inputs when focused                                                            |
-| `formBorderRadius`        | `--form-border-radius`         | `20px`                                    | `20px`                                    | Border radius style for the form                                                                      |
-| `formContentBorderRadius` | `--form-content-border-radius` | `6px`                                     | `6px`                                     | Border radius style for form content (for example: inputs, buttons)                                   |
 
-Here is an example of customizing only the background color for the light theme using the Feedback constructor configuration:
+| `triggerBackground`       | `--trigger-background`         | `var(--button-background)`                | `var(--button-background)`                | Background color of the widget (injected button and form)                                             |
+| `triggerBackgroundHover`  | `--trigger-background-hover`   | `var(--button-background-hover)`          | `var(--button-background-hover)`          | Background color of the injected button when in a hover state                                         |
+| `triggerBorderRadius`     | `--trigger-border-radius`      | `12px` TODO                               | `12px` TODO                               | Border radius style used for the widget (injected button and success message)                         |
+
+| `dialogBackground`        | `--dialog-background`          | `var(--background)`                       | `var(--background)`                       | Background color of the widget (injected button and form)                                             |
+| `dialogBorderRadius`      | `--dialog-border-radius`       | `20px`                                    | `20px`                                    | Border radius style for the form                                                                      |
+
+
+It's possible to set the `id` configuration value to something
```

Comment:
Did you mean to add more to this sentence?

Reply:
derp. fixed up this area.